health/notifications/health_alarm_notify.conf

```diff
@@ -284,6 +284,7 @@ SEND_OPSGENIE="YES"
 
 # Api key
 OPSGENIE_API_KEY=""
+OPSGENIE_API_URL=""
```

Comment:
This should probably be explicitly set to a sane default if possible.

Reply:
I arrived in the same conclusion when I saw the PR #10562.

@tinyhammers , please ,add the previous URL as default value here.

Reply:
Sensible. Set to https://api.opsgenie.com